MAINTAINERS.yml

```diff
@@ -764,6 +764,17 @@ Release Notes:
   labels:
     - "area: FPGA"
 
+"Drivers: Fuel Gauge":
+  status: maintained
+  maintainers:
+    - aaronemassey
+    - teburd
+  files:
+    - drivers/fuel_gauge/
+    - dts/bindings/fuel-gauge/
+    - include/zephyr/drivers/fuel_gauge.h
+    - tests/drivers/fuel_gauge/
```

Comment:
Please add `area: Fuel Gauge` label as well.

Reply:
Done. Apologies for the delay as I was sick yesterday.